conversions/Backwards/src/main/java/com/agentdid127/resourcepack/backwards/impl/DeleteFileConverter.java

```diff
@@ -71,14 +67,12 @@ public void deleteBlocks(int version) throws IOException {
             if (version < Util.getVersionProtocol(packConverter.getGson(), "1.13")) {
                 blockMPath = models.resolve("block");
                 blockTPath = textures.resolve("blocks");
-            }
-            else {
+            } else {
```

Comment:
Else needs to be here

Reply:
it is there, it just was moved to one line instead of being on a seperate line